src/careamics/lvae_training/get_config.py

```diff
@@ -6,7 +6,7 @@
 
 import ml_collections
```

Comment:
ml_collections still in use?

Reply:
There are still ml_config parameters that are used in the training logic. I tried only to do the dataset params in this PR and then see what we can do next when @federico-carrara 's training code is ready.

Reply:
I see, makes sense